editor/package-lock.json

```diff
@@ -1199,15 +1199,184 @@
       }
     },
     "@babel/plugin-transform-modules-commonjs": {
-      "version": "7.10.1",
-      "resolved": "https://registry.npmjs.org/@babel/plugin-transform-modules-commonjs/-/plugin-transform-modules-commonjs-7.10.1.tgz",
-      "integrity": "sha512-AQG4fc3KOah0vdITwt7Gi6hD9BtQP/8bhem7OjbaMoRNCH5Djx42O2vYMfau7QnAzQCa+RJnhJBmFFMGpQEzrg==",
-      "dev": true,
+      "version": "7.10.4",
```

Comment:
what's up with this guy? you didn't change the package.json

Reply:
#580 came with a `package.json` change but the corresponding `package-lock.json` wasn't committed